pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -91,11 +92,27 @@ var rebuildContainerTpl = template.Must(
 				`)[1:], // remove leading newline
 	))
 
-type upload struct {
+// transfer describes the source and destination of either a move or copy.
+type transfer struct {
 	From string
 	To   string
 }
 
+var rebuildCleanupTpl = template.Must(
```

Comment:
The one "cleanup" thing it's definitely doing is the tetragon teardown. Otherwise it could maybe be called rebuildExport or rebuildDataDump or some other extraction verb.

How about rebuildFinalize?

Reply:
yep i like that